ansible_bender/cli.py

```diff
@@ -16,6 +16,7 @@
 from ansible_bender.db import PATH_CANDIDATES
 from ansible_bender.okd import build_inside_openshift
 
+from utils import fancy_time
```

Comment:
```suggestion
from ansible_bender.utils import fancy_time
```
Otherwise it raises `ModuleNotFoundError: No module named 'utils'`